src/gui/owncloudsetupwizard.cpp

```diff
@@ -308,6 +308,16 @@ void OwncloudSetupWizard::testOwnCloudConnect()
     connect(job, &PropfindJob::result, _ocWizard, &OwncloudWizard::successfulStep);
     connect(job, &PropfindJob::finishedWithError, this, &OwncloudSetupWizard::slotAuthError);
     job->start();
+
+    // get the display name so that the folder we register with the file browser has a pretty name
+    auto userJob = new JsonApiJob(account, QStringLiteral("ocs/v2.php/cloud/user"), this);
+    connect(userJob, &JsonApiJob::jsonReceived, this, [account](const QJsonDocument &json, int status) {
+        if (status == 200) {
+            const QString user = json.object().value(QStringLiteral("ocs")).toObject().value(QStringLiteral("data")).toObject().value(QStringLiteral("display-name")).toString();
```

Comment:
does this need any check in the chain or at the end if user is empty?

Reply:
if its our server and we got a 200 its really supposed to be there and if we didn't get a 200 well probably the first check failed and nothing is lost